### DISCUSSION TITLE
How to specify Haskell project in subfolder of flake root?

### DISCUSSION THREAD

**Discussion by malteneuss** [srid/haskell-flake#191] [Q&A] [answered]
I would like to use haskell-flake on NixOS, because it looks way easier to setup than the other Nix integrations; i have a project where the cabal file (and the rest of the Haskell project) is in a subfolder and i can't find a `haskell-flake` attribute to declare that. Is such a setup possible?

**Comment by srid** [comment, marked as answer]
You just create a `cabal.project` file with `packages` option https://github.com/srid/haskell-multi-nix/blob/master/cabal.project
